generators/csharp/sdk/versions.yml

```diff
@@ -7,6 +7,20 @@
 #         Set `enable-forward-compatible-enums` to `false` in the configuration to generate the old enums.
 #   irVersion: 53
 
+- version: 1.10.00
+  createdAt: "2025-02-21"
+  irVersion: 53
+  changelogEntry:
+    - type: feat
+      summary: |
+        Add support for generating dates. Configure the type of dates you generating using the `date-type` configuration option.
+        `date-type` has the following options:
+        - `use-date-only-portable` (default): Generates native `DateOnly` properties on .NET 6 and above, and falls back to using the third party `Portable.System.DateTimeOnly` package.
+        - `use-date-time`: Generates dates as `DateTime` properties.
+        - `use-date-only-on-net6+`: Generates native `DateOnly` properties on .NET 6 and above, and falls back to using `DateTime` on older versions of .NET.
```

Comment:
Couple thoughts on the naming of this configuration:
1. The `use` prefix is redundant across every setting - I vote we drop it.
2. We could make it more clear that there is a fallback behavior.

I'd say we should have:

```yaml
date-type: date-only-with-portable-fallback
```

```yaml
date-type: date-only-with-date-time-fallback
```

```yaml
date-type: date-time
```

It's still a mouthful, but I think it's a little more clear to the user. Let me know what you think though - open to ideas here.

Reply:
That sounds better.